model/security/get_action_groups/structures.smithy

```diff
@@ -0,0 +1,16 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetActionGroups_Input {}
+
+@output
+structure GetActionGroups_Output {
+    actiongrouplist: ActionGroupsList
```

Comment:
This operation returns a map:
```suggestion
    @httpPayload
    content: ActionGroupMap
```

It is expected that this will cause build errors until #127 is merged